packages/@glimmer/syntax/test/loc-node-test.ts

```diff
@@ -134,6 +134,29 @@ test('html elements', () => {
   }
 });
 
+test('html elements with paths', () => {
+  let ast = parse(`
+    <Foo as |bar|>
+      <bar.x.y class='bar'/>
+      <bar.x.y class='bar'></bar.x.y>
+    </Foo>
+  `);
+
+  let [, foo] = ast.body;
+  locEqual(foo, 2, 4, 5, 10, 'Foo element');
```

Comment:
is this range right? 4 to 10 is 6, but the element is `<Foo as |bar|>` ~ 14

Reply:
the end is at `</Foo>`, so this is correct

Reply:
what is the range of 6 though? feels ... arbitrary? I'm probably missing something though.

Reply:
start: line 2, column 4
end: line 5, column 10

different line

Reply:
lol oops. thanks